src/document_store.zig

```diff
@@ -248,6 +252,8 @@ fn uriFromImportStr(store: *DocumentStore, allocator: *std.mem.Allocator, handle
 }
 
 pub const AnalysisContext = struct {
+    const Self = @This();
```

Comment:
Minor style point, I don't think this is needed, you can use `AnalysisContext` in `clone`

Reply:
Fixed